code/modules/mob/living/silicon/pai/recruit.dm

```diff
@@ -334,7 +337,7 @@ var/datum/paiController/paiController			// Global handler for pAI candidates
 						</tr>
 					</table>
 					<table class="download">
-						<td class="download"><a href='byond://?src=[UID()];download=1;candidate=\ref[c];device=\ref[p]' class="button"><b>Download [c.name]</b></a>
+						<td class="download"><a href='byond://?src=[UID()];download=1;candidate=\ref[c];device=\ref[p];downloader=\ref[user]' class="button"><b>Download [c.name]</b></a>

```

Comment:
\ref doesn't function in 512 like it did in 511. We should be using the replacement instead.

Reply:
what is the replacement?

Reply:
I don't think you need to use \ref in the first place. Simply use usr